### PR TITLE
Add support for #[no_std]

### PR DESCRIPTION
**Pull request by tomaka**
I've taken the liberty to bump the version in the same PR.

(note: this is for a personal project not parity-related)

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -8,7 +8,12 @@
 // at https://www.apache.org/licenses/LICENSE-2.0 and a copy of the MIT license
 // at https://opensource.org/licenses/MIT.
 
-use std::{hash::{BuildHasherDefault, Hasher}, marker::PhantomData};
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(feature = "std")]
+extern crate core;
```
Comment (twittner) [inline src/lib.rs]: We do not need this `extern crate core;` I think.

Comment (tomaka) [reply src/lib.rs]: It doesn't compile without.

Comment (twittner) [reply src/lib.rs]: Sorry,  the `Cargo.toml` needs an `edition = "2018"`.

Comment (tomaka) [reply src/lib.rs]: Done

**Review by twittner — APPROVED** [review]

**Pull request merged**